runtime/vm/visible.c

```diff
@@ -85,6 +85,12 @@ checkModuleAccess(J9VMThread *currentThread, J9JavaVM* vm, J9ROMClass* srcRomCla
  */
 IDATA
 checkVisibility(J9VMThread *currentThread, J9Class* sourceClass, J9Class* destClass, UDATA modifiers, UDATA lookupOptions)
+{
+	return checkVisibilityModuleAccessOption(currentThread, sourceClass, destClass, modifiers, lookupOptions, TRUE);
```

Comment:
instead of this can we just introduce a `J9_LOOK_NO_MODULARITY_FLAG` ?

Reply:
i just changed the name, last one didnt really make sense

Reply:
Good idea, use `J9_LOOK_NO_MODULE_CHECKS` instead. Please have another look.